src/test/java/com/google/api/codegen/testdata/discoveries/go/go_bigquery.v2.json.baseline

```diff
@@ -20,7 +20,7 @@ import (
 
 func main() {
   ctx := context.Background()
-  httpClient, err := google.DefaultClient(ctx, "https://www.googleapis.com/auth/cloud-platform")
+  httpClient, err := google.DefaultClient(ctx, bigquery.BigqueryScope)
```

Comment:
It might be preferable, but we can't use it anyway. The published client libraries use BigqueryScope (lowercase 'q'). As far as I am concerned, this is a feature not a bug.